src/Mod/Draft/draftfunctions/rotate.py

```diff
@@ -141,10 +141,8 @@ def rotate(objectslist, angle, center=App.Vector(0,0,0),
                     g = newgroups.setdefault(p.Name, App.ActiveDocument.addObject(p.TypeId, p.Name))
                     g.addObject(newobj)
                     break
-    if copy and utils.getType("selectBaseObjects"): # was utils.getType("selectBaseObjects", False)
-        gui_utils.select(objectslist)
-    else:
-        gui_utils.select(newobjlist)
+
+    gui_utils.select(newobjlist)
```

Comment:
I noticed that this doesn't work in the current code, so I submitted #3668 to fix it. However, I think with this pull request my fix is unnecessary.